lib/api/dsl/storage/index.js

```diff
@@ -503,9 +523,7 @@ function destroy(obj, index, collection, field) {
  */
 function reindexTestTable(filters, subfilters, testTables, index, collection) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Function has a complexity of 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AFunctionComplexity)